src/main/java/task/ToDo.java

```diff
@@ -0,0 +1,31 @@
+package task;
+
+public class ToDo extends Task {
+    public ToDo(boolean isDone, String description) {
+        super(isDone, description);
+    }
+
+    @Override
+    public String toFileString() {
+        StringBuilder sb = new StringBuilder("T | ");
+        if (isDone) {
+            sb.append("1 | ");
+        } else {
+            sb.append("0 | ");
+        }
+        sb.append(description);
+        return sb.toString();
+    }
+
+    @Override
+    public String toString() {
+        StringBuilder sb = new StringBuilder("[T]");
+        if (isDone) {
+            sb.append("[✓] ");
+        } else {
+            sb.append("[✗] ");
+        }
```

Comment:
a ternary statement here might be more readable

Reply:
Appreciate it, but it would require further refactoring as in this instance as I am not returning a value.

Reply:
try `sb.append( isDone ? "[✓] " : "[✗] " );`
I think that this should work